### PR TITLE
add Naga Initiate enemy, nagawarrior sprite/atlas/deck

### PR DESCRIPTION
**Pull request by schnautzr**
Deck is by MasterXploder7.
Sprite is an original derivative of HAS Creature Pack naga0.
The green/blue challenger Naga Initiate is added to the white biome.

### CODE REVIEW

File: forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas [hunk 1]
```diff
@@ -0,0 +1,68 @@
+nagawarrior.png
+size: 64,96
```
Comment (Simisays) [inline forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas:2]: wrong size, causes crash with the adventure editor (which is a great tool to check sprite animations) 

64,80

File: forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas [hunk 2]
```diff
@@ -0,0 +1,68 @@
+nagawarrior.png
+size: 64,96
+format: RGBA8888
+filter: Nearest,Nearest
+repeat: none          
+Avatar
+  xy: 0, 0
+  size: 16, 16
```
Comment (Simisays) [inline forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas:8]: no custom or dedicated avatar ?

File: forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas
Comment (Simisays) [inline forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas:1]: y raster seems wrong 

Y:0 (idle)
Y:16 (walk)
Y;32 (attack or hit ?)
Y:48 (attack or hit ?)
Y:64 (death)

File: forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas (hunk 1 above)
Comment (schnautzr) [reply forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas:2]: Thanks, fixing.

File: forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas (hunk 2 above)
Comment (schnautzr) [reply forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas:8]: I knew I'd forgotten something!

File: forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas
Comment (schnautzr) [reply forge-gui/res/adventure/Shandalar/sprites/naga/nagawarrior.atlas:1]: Will check.

**Comment by schnautzr** [comment]
Added the missing line with the avatar. This is why I should go to bed earlier.

**Review by Simisays — APPROVED** [review]
:+1:

**Pull request merged**